guacamole/src/main/java/org/apache/guacamole/rest/auth/AuthenticationService.java

```diff
@@ -328,6 +354,82 @@ private List<DecoratedUserContext> getUserContexts(GuacamoleSession existingSess
         return userContexts;
 
     }
+    
+    /**
+     * Resumes authentication using given credentials if a matching resumable 
+     * state is found.
+     *
+     * @param credentials 
+     *     The initial credentials containing the request object.
+     *
+     * @return 
+     *     Resumed credentials if a valid resumable state is found; otherwise, 
+     *     returns {@code null}.
+     */
+    private Credentials resumeAuthentication(Credentials credentials) {
+        
+        Credentials resumedCredentials = null;
+
+        // Retrieve signed State from the request
+        HttpServletRequest request = credentials.getRequest();
+        
+        // Retrieve the provider id from the query parameters.
+        String resumableProviderId = request.getParameter(Credentials.RESUME_QUERY);
+        // Check if a provider id is set.
+        if (resumableProviderId == null || resumableProviderId.isEmpty()) {
+            // return if a provider id is not set.
```

Comment:
The formatting of these single-line comments is a bit funky - comments like this should always start with a capital letter (this one doesn't), and generally do not end with a period.

This comment should definitely be capitalized. I'd also consider removing the periods to match existing comment style.